test/testdata/NotificationTestData.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package testdata
+
+import java.time.Instant
+
+import models.UcrBlock
+import models.notifications.{Entry, NotificationFrontendModel, ResponseType}
+import testdata.CommonTestData.{conversationId, correctUcr}
+
+object NotificationTestData {
+
+  def exampleNotificationFrontendModel(
```

Comment:
`notification(...)`

Reply:
As explained earlier, the type returned is `NotificationFrontendModel`, so method name should match it.